Libraries/Modal/Modal.js

```diff
@@ -45,16 +39,27 @@ export type Props = $ReadOnly<{|
   ...ViewProps,
 
   /**
-   * The `animationType` prop controls how the modal animates.
-   *
-   * See https://reactnative.dev/docs/modal.html#animationtype
+    The `animationType` prop controls how the modal animates.
+
+    - `slide` slides in from the bottom
+    - `fade` fades into view
+    - `none` appears without an animation
+
+    Default is set to `none`.
```

Comment:
Should be used with default annotation.

Reply:
Agreed

Reply:
Okay, let's leave it the way it is for now.